packages/builder/src/components/design/settings/controls/FieldConfiguration/FieldSetting.svelte

```diff
@@ -49,6 +50,7 @@
       {bindings}
       {parseSettings}
       on:change
+      disabled={updating}
```

Comment:
We can remove the `disabled` prop now since we can also remove this `updating` prop.